<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIWindow</class>
 <widget class="QMainWindow" name="UIWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>755</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UIWindow</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="listCloud">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>256</width>
      <height>711</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1233</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="file"/>
   </widget>
   <widget class="QMenu" name="menuClustering">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Clustering</string>
    </property>
    <addaction name="actionMST"/>
    <addaction name="actionCLOPE"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuClustering"/>
  </widget>
  <action name="file">
   <property name="text">
    <string>Open new cloud</string>
   </property>
  </action>
  <action name="actionMST">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MST</string>
   </property>
  </action>
  <action name="actionCLOPE">
   <property name="text">
    <string>CLOPE</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
